packages/pds/src/app-view/api/app/bsky/feed/getPostThread.ts

```diff
@@ -275,32 +316,39 @@ class ParentNotFoundError extends Error {
 // READ AFTER WRITE
 // ----------------
 
-const ensureReadAfterWrite = async (
+const getPostThreadMunge = async (
   ctx: AppContext,
-  requester: string,
-  res: ApiRes<OutputSchema>,
+  original: OutputSchema,
+  local: LocalRecords,
 ): Promise<OutputSchema> => {
-  const rev = getRepoRev(res.headers)
-  if (!rev) return res.data
-  if (!isThreadViewPost(res.data.thread)) {
-    return res.data
-  }
-  let thread: ThreadViewPost = res.data.thread
-  const localSrvc = ctx.services.local(ctx.db)
-  const local = await localSrvc.getRecordsSinceRev(requester, rev, [
-    ids.AppBskyFeedPost,
-  ])
-  const inThread = findPostsInThread(thread, local.posts)
-  if (inThread.length < 0) return res.data
-  for (const record of inThread) {
-    thread = await insertIntoThreadReplies(localSrvc, thread, record)
+  if (!isThreadViewPost(original.thread)) {
+    return original
   }
```

Comment:
I suppose technically if `original.thread` is a `NotFoundPost` we might consider handling it similarly to a not-found error.  But I am comfortable just leaving a note about that and moving on, since in practice it doesn't happen on our service right now.

Reply:
yup good call - left a note